packages/schema/docs/build/schema.md

```diff
@@ -1059,12 +1059,12 @@ How will we get a list of new objects? Will be turned into a trigger automatical
 
 * ```
   { display: { label: 'New User', description: 'Trigger when a new User is created in your account.' },
-    operation:
+    operation: 
```

Comment:
it's maybe not worth your time to fix, but might be nice to solve why these keep getting added.

Reply:
I finally spent some time looking into this. It's because `util.inspect` generates slightly different outputs. On Node 8, which I'm using, it outputs trailing spaces, while on Node 12, which you're probably using, it doesn't generate trailing spaces.

Added some code to ensure a consistent behavior in 72c6f77.

Reply:
ah yeah, i'm mostly on 10 or 12. that'll do it! Thanks for digging in.